ocrd_tesserocr/ocrd-tool.json

```
@@ -202,7 +202,81 @@
           "default": "DEFAULT",
           "description": "Tesseract OCR engine mode to use:\n* Run Tesseract only - fastest,\n* Run just the LSTM line recognizer. (>=v4.00),\n*Run the LSTM recognizer, but allow fallback to Tesseract when things get difficult. (>=v4.00),\n*Run both and combine results - best accuracy."
         }
-      }
+      },
+      "resource_locations": ["module"],
+      "resources": [
+        {
+          "url": "https://ub-backup.bib.uni-mannheim.de/~stweil/ocrd-train/data/Fraktur_5000000/tessdata_fast/Fraktur_50000000.334_450937.traineddata",
+          "name": "Fraktur_GT4HistOCR.traineddata",
+          "parameter_usage": "without-extension",
+          "description": "Tesseract LSTM model trained on GT4HistOCR",
+          "size": 1058487
+        },
+        {
+          "url": "https://ub-backup.bib.uni-mannheim.de/~stweil/ocrd-train/data/ONB/tessdata_best/ONB_1.195_300718_989100.traineddata",
+          "name": "ONB.traineddata",
+          "parameter_usage": "without-extension",
+          "description": "Tesseract LSTM model based on Austrian National Library newspaper data",
+          "size": 4358948
+        },
+        {
```

more up-to-date model:
```suggestion
        {
        "url": "https://ub-backup.bib.uni-mannheim.de/~stweil/tesstrain/frak2021/tessdata_best/frak2021-0.905.traineddata",
          "name": "frak2021.traineddata",
          "parameter_usage": "without-extension",
          "description": "Tesseract LSTM model based on Austrian National Library newspaper data",
          "size": 3421140
        },
        {
```